frontends/web/src/components/balance/balance.tsx

```diff
@@ -15,22 +15,20 @@
  * limitations under the License.
  */
 
-import { PropsWithChildren } from 'react';
+import { FunctionComponent } from 'react';
+import { useTranslation } from 'react-i18next';
 import { IBalance } from '../../api/account';
 import { FiatConversion } from '../../components/rates/rates';
-import { translate, TranslateProps } from '../../decorators/translate';
 import style from './balance.module.css';
 
-interface BalanceProps {
+interface Props {
     balance?: IBalance;
 }
```

Comment:
@BrianCraig if you touch it anyway, would you mind changing to  `type Props = {` ?